Assignment-2/Ques6.js

```diff
@@ -0,0 +1,13 @@
+
+// Write a function expression that takes in a number and returns true if it's even and false if it's odd.
+
+function checkEven(number){
+    if(number%2==0){
```

Comment:
Test for negative numbers, 0 , 1, 2 as well. Remove consoles. Rename function name to more understandable